demo/gdi/nuklear_gdi.h

```diff
@@ -62,28 +74,32 @@ nk_create_image(struct nk_image * image, const char * frame_buffer, const int wi
 {
     if (image && frame_buffer && (width > 0) && (height > 0))
     {
-        image->w = width;
-        image->h = height;
+        image->w = (unsigned short)width;
+        image->h = (unsigned short)height;
         image->region[0] = 0;
         image->region[1] = 0;
-        image->region[2] = width;
-        image->region[3] = height;
+        image->region[2] = (unsigned short)width;
+        image->region[3] = (unsigned short)height;
         
         INT row = ((width * 3 + 3) & ~3);
-        BITMAPINFO bi = { 0 };
+        BITMAPINFO bi;
+        memset( &bi, 0, sizeof(bi) );
```

Comment:
Is this required? It's less efficient and slightly "cumbersome" compared to the previous version.

Reply:
This is not required. 
The compiler got warnings because "0" is not equal "unsinged long".
If you changed the "0" to "0ul" the compiler said "0ul" is not equal to "int". 
I think the notation "{ 0 }" is not really designed for structs with different data types?

If you thinks its better to reverse this. Just do it.

Reply:
Hm, that's really weird. According to C99 and newer, the notation `{ 0 }` denotes the minimal case for partial struct assignment which in this case means assign `0` to the first struct member and all following struct members disregarding their type (which might be potentially harmful as e.g. `NULL` can be something else than `0` on some platforms or some IEEE 754 implementations could behave incorrectly if some flags are zero, but programmers for such platforms would know what to do with this source :wink:).

Are you using the provided `build.bat` script? Or some other compiler? What are all the options?

I would like to find out what's the issue in this case as it sounds really suspicious (might be also an issue with Microsoft Visual Studio compiler as used in `build.bat`).